ci/workflows/smoke.workflow

```diff
@@ -0,0 +1,28 @@
+workflow "smoke tests" {
+  resolves = "run test"
+}
+
+action "build ceph and skyhook cls" {
+  uses = "popperized/cmake@ubuntu-18.04"
+  args = "cls_tabular run-query ceph_test_skyhook_query vstart"
```

Comment:
Hmmm... `make vstart` builds the OSD, MON, and a bunch of other stuff ([Ceph src/Makefile](https://github.com/ceph/ceph/blob/master/src/CMakeLists.txt#L744)). I think this takes too long. I got around this by pulling the `skyhook-test` container, which had these binaries pre-built... 

Not saying my solution is correct (I actually don't like it) -- but it works... :(